.env

```diff
@@ -0,0 +1 @@
+SECRET_KEY=40341b8c51e75263efc9ca40f05bb6132255d3399c7804c46dcf6892e364081d
```

Comment:
Each instance should have its own unique key, so the `.env` file should not be kept under source control.  We could however provide a sample file or provide an example directly in the documentation (README), such as:

```
# Generate with openssl rand -hex 32
SECRET_KEY=
```

Reply:
The new commit has a `.env` file with `SECRET_KEY=`, which is fine except that the file is still under source control.  So when a user edits it, it will be seen as a change by Git.  This can be a problem as users are likely to commit their secret key or they will always have a change never committed in this file.

I guess a simple way to avoid this issue is to rename this `.env` file as `env.sample` and mention in the README that it needs to be copied to `.env` to be loaded by docker-compose.  It's probably also worth adding `.env` to the `.gitignore` file to be sure it's never added in git.